doc/source/rllib/rllib-torch2x.rst

```diff
@@ -0,0 +1,134 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+
+
+Using RLlib with torch 2.x compile
+==================================
+
+Torch 2.x comes with the new ``torch.compile()`` `API <https://pytorch.org/docs/stable/generated/torch.compile.html#torch.compile>`_, which leverages `torch dynamo <https://pytorch.org/docs/stable/dynamo/index.html#torchdynamo-overview>`_ under the hood to JIT-compile wrapped code. We integrate ``torch.compile()`` with RLlib in the context of `RLModules <rllib-rlmodule.html>`_ and Learners. 
+
+We have integrated this feature with RLModules. You can set the backend and mode via ``framework()`` API on an :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig` object. Alternatively, you can compile the :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule` directly during stand-alone usage such as inference.
+
+Benchmarks
+==========
+
+We conducted a comprehensive benchmark with this feature. 
+
+Inference
+---------
+For the benchmarking metric, we compute the inverse of the time it takes to run :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration` of the RLModule. We have conducted this benchmark on the default implementation of PPO RLModule under different hardware settings, torch versions, dynamo backends and modes, as well as different batch sizes. Here is a high-level summary of our findings:
```

Comment:
To make this hyper-accessible, we should point to the `TorchRLModule.compile()` API for inference.